docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
+| Option                       | Summary                                                                                                 | Default  |
++==============================+=========================================================================================================+==========+
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``land_option``               | Switch to choose land option as described above.                                                        | ``none`` |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``land_depth``                | Value of land mixed layer depth.                                                                        | ``-1``   |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``slandlon``                  | Vector determining lower bounds of longitudes for land masses.                                          | ``0``    |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``slandlat``                  | Vector determining lower bounds of latitudes for land masses.                                           | ``0``    |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``elandlon``                  | Vector determining higher bounds of longitudes for land masses.                                         | ``-1``   |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``elandlat``                  | Vector determining higher bounds of latitudes for land masses.                                          | ``-1``   |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``land_h_capacity_prefactor`` | Factor by which to multiply ocean heat capacity to get land heat capacity if ``input`` option is used.  | ``1.0``  |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+|``land_albedo_prefactor``     | Factor by which to multiply ocean albedo to get land albedo if ``input`` option is used.                | ``1.0``  |
++------------------------------+---------------------------------------------------------------------------------------------------------+----------+
+
+Other options (temporary title)
```

Comment:
Title update needed.You missed the following options:
trop_depth, trop_cap_limit, heat_cap_limit, np_cap_factor, flux_lhe_anom_file_name, flux_lhe_anom_field_name

Reply:
Did you decide not to include these options or was this missed?

Reply:
Sorry only now saw your response from 9 Dec. No problem to not include them.